src/main/resources/userEmailTemplate-feedbackSessionClosed.html

```diff
@@ -0,0 +1,13 @@
+Hello ${userName},
+
+The following feedback session is closed.
```

Comment:
`This is a gentle reminder sent to instructors to let you know that the following session is now closed.`